### PR TITLE
Compile constants checked with assertion

### PR DESCRIPTION
**Pull request by craff**
constants in Events end EventFd are now compile time constants

### CODE REVIEW

**Comment by craff** [comment]
I did this PR which is ok for me. Another solution is to generate an ML file from a small c program. 
The advantage is that the current check is a runtime check and you could still deploy wrong code. I am submitting a PR constant 3 and you can choose.

File: lib/polly.ml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+let assert_eq x y =
+  if x <> y then (
+    Printf.eprintf "0x%x != 0x%x = 0o%o\n%!" x y y ;
```
Comment (lindig) [inline lib/polly.ml:3]: If you feel that using `assert (out = polly_OUT ())` is not enough but prefer to use your own assertion, I would:

* pass a third argument for the location, which is `__LOC__` at the caller
* think libraries should not use stderr but use the exception mechanism to communicate problems
* would use the `Failure` exception with a string argument

**Comment by craff** [comment]
You do not prefer the other PR? This one with assertion will only fail at runtime. 

Le 5 août 2023 23:17:41 GMT-10:00, Christian Lindig ***@***.***> a écrit :

**Comment by craff** [comment]
close because #10 is better

**Pull request closed without merging**